openapi/generator.go

```diff
@@ -345,6 +347,10 @@ func isResponseCodeRange(code string) bool {
 // setOperationResponse adds a response to the operation that
 // return the type t with the given media type and status code.
 func (g *Generator) setOperationResponse(op *Operation, t reflect.Type, code, mt, desc string, headers []*ResponseHeader) error {
+	return g.setOperationResponseV2(op, t, code, mt, desc, headers, nil, nil)
```

Comment:
This is, of course, a temporary hack. For sure, this would be fixed, if we decide to proceed with this PR.